spec/uploader_spec.rb

```diff
@@ -385,6 +385,21 @@
     it "should fail if timeout is reached" do
       expect{Cloudinary::Uploader.upload(Pathname.new(TEST_IMG), :tags => [TEST_TAG, TIMESTAMP_TAG])}.to raise_error(RestClient::RequestTimeout)
     end
+
+    it "should allow passing nil value" do
+      expect(RestClient::Request).to receive(:execute) do |args|
+        expect(args[:timeout]).to be nil
+      end
+      Cloudinary::Uploader.upload(TEST_IMG, :timeout => nil)
+    end
+
+    it "should fall back to default timeout" do
+      Cloudinary.config.delete_field(:timeout)
```

Comment:
Maybe want to reset the state of `Cloudinary.config` after the test?

Reply:
this test is under   `describe ":timeout" do` on line 376